pkg/virt-launcher/virtwrap/network/common.go

```diff
@@ -198,9 +198,9 @@ func (h *NetworkUtilsHandler) SetRandomMac(iface string) (net.HardwareAddr, erro
 		if changed {
 			mac, err = Handler.GetMacDetails(iface)
 			if err != nil {
-				log.Log.Reason(err).Errorf("updated MAC for interface: %s - %s -> %s", iface, currentMac, mac)
 				return nil, err
 			}
+			log.Log.Reason(err).Infof("updated MAC for interface: %s - %s -> %s", iface, currentMac, mac)
```

Comment:
'Reason(err)' is wrong. That is one of the reasons why we moved it to the error branch. Can you make the message more descriptive? Was there before but without looking at the source code it would be impossible for me to understand its meaning.

Reply:
It shouldn't be an error at all, missed that. Sec..